README.md

```diff
@@ -4,4 +4,120 @@ The stilk of an apple is also called a pedicel.
 
 This Ruby gem will help you handle an Apple Pay `PKPaymentToken`.
 
+## Usage
 
+```ruby
+# Parse the payment token into a hash.
+apple_pay_payment_token_hash = JSON.parse(apple_pay_payment_token_json)
+
+# Instantiate.
+# Validates the format of the Apple Pay payment token; raises an
+# Pedicel::Validator::Error if the format is invalid.
+pedicel = Pedicel::EC.new(apple_pay_payment_token_hash)
+
+# Decrypt using the symmetric key directly.
+# The symmetric key `sk` is derived from the private key and either the
+# merchant ID directly or the merchant certificate.
+data = pedicel.decrypt(symmetric_key: sk)
+
+# Decrypt using the merchant private key and merchant certificate.
+# The merchant private key `pk` and merchant certificate `c` are generated
+# during the Apple Pay merchant sign-up.
+data = pedicel.decrypt(private_key: pk, certificate: c)
+
+# Decrypt using the merchant private key and merchant ID.
+# The merchant private key `pk` and merchant ID `mid` are generated during the
+# Apple Pay merchant sign-up.
+data = pedicel.decrypt(private_key: pk, merchant_id: mid)
+
+# Validate the decrypted data; raises an Pedicel::Validator::Error if the
+# format is invalid.
+Pedicel::Validator.validate_token_data(data)
```

Comment:
Actually not, because this is a validation of the decrypted token data whereas #14 introduces automatic validation of the inputted (unencrypted) token.

Reply:
Minor: I'd use the term _Payment Data_ somewhere in the comment, it's what Apple calls the data. As above, you could also link to what tests are being performed.